dbux-common/src/util/TriggerablePromise.js

```diff
@@ -0,0 +1,55 @@
+import sleep from './sleep';
+
+export default class TriggerablePromise {
```

Comment:
rename to `SyncPromise` . These are generally used to let outside threads/actors wait for/synchronize against an asynchronous operation.